framework/src/main/java/org/checkerframework/common/value/ValueAnnotatedTypeFactory.java

```diff
@@ -1302,6 +1302,41 @@ private boolean methodIsStaticallyExecutable(Element method) {
 
         @Override
         public Void visitMethodInvocation(MethodInvocationTree tree, AnnotatedTypeMirror type) {
+            if (getMethodIdentifier().isMathMin(tree, processingEnv)
+                    && type.hasAnnotation(UNKNOWNVAL)) {
+                AnnotatedTypeMirror argType1 = getAnnotatedType(tree.getArguments().get(0));
+                AnnotatedTypeMirror argType2 = getAnnotatedType(tree.getArguments().get(1));
+                if (argType1.hasAnnotation(IntRange.class)
+                        && argType2.hasAnnotation(IntRange.class)) {
+                    long from =
+                            Math.min(
+                                    getFromValueFromIntRange(argType1),
+                                    getFromValueFromIntRange(argType2));
```

Comment:
Assignments like this can probably be split because they don't look well.

Reply:
They do not. I suggest pulling the calls to `getFromValueFromIntRange` and `getToValueFromIntRange` out into locals.

I would also move all of this logic into a private helper method.